python/pyproject.toml

```diff
@@ -15,15 +15,14 @@
 [build-system]
 build-backend = "setuptools.build_meta"
 requires = [
-    "wheel",
-    "setuptools>=61.0.0",
-    "cython>=0.29,<0.30",
-    "scikit-build>=0.13.1",
     "cmake>=3.23.1,!=3.25.0",
+    "cython>=0.29,<0.30",
     "ninja",
-    "cuda-python>=11.7.1,<12.0",
-    "tomli; python_version < '3.11'",
-]
+    "scikit-build>=0.13.1",
+    "setuptools",
```

Comment:
Looks like the lower bound was dropped. Added a suggestion above related to addressing this

Reply:
Fixed, thanks.